mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -364,29 +366,37 @@ def __call__(self, datasets):
         """
         self.ndatasets = len(datasets)
         params = self.params
-        _shpaldebug("SearchlightHyperalignment %s for %i datasets" % (self, self.ndatasets))
+
+        _shpaldebug("SearchlightHyperalignment %s for %i datasets"
+                    % (self, self.ndatasets))
         if params.ref_ds in params.exclude_from_model:
-            raise (ValueError, "Requested reference dataset %i is also "
-                               "in the exclude list." % params.ref_ds)
+            raise ValueError("Requested reference dataset %i is also "
+                             "in the exclude list." % params.ref_ds)
         if params.ref_ds != params.hyperalignment.params.ref_ds:
             warning('Supplied ref_ds & hyperalignment instance ref_ds:%d differ.'
                     % params.hyperalignment.params.ref_ds)
             warning('Using default hyperalignment instance with ref_ds: %d' % params.ref_ds)
             params.hyperalignment = Hyperalignment(ref_ds=params.ref_ds)
         if params.ref_ds >= self.ndatasets:
-            raise (ValueError, "Requested reference dataset %i is out of "
-                               "bounds. We have only %i datasets provided" % (params.ref_ds, self.ndatasets))
+            raise ValueError("Requested reference dataset %i is out of "
+                             "bounds. We have only %i datasets provided"
+                             % (params.ref_ds, self.ndatasets))
         if len(params.exclude_from_model) > 0:
-            warning("These datasets will not participate in building common model: %s" % params.exclude_from_model)
+            warning("These datasets will not participate in building common "
+                    "model: %s" % params.exclude_from_model)
+
         # Setting up SearchlightHyperalignment
         # we need to know which original features where comprising the
         # individual SL ROIs
         _shpaldebug('Initializing HyperalignmentMeasure.')
-        hmeasure = HyperalignmentMeasure(featsel=params.featsel,
-                                         hyperalignment=params.hyperalignment,
-                                         full_matrix=params.combine_neighbormappers,
-                                         use_same_features=params.use_same_features,
-                                         exclude_from_model=params.exclude_from_model, dtype=params.dtype)
+        hmeasure = HyperalignmentMeasure(
+            featsel=params.featsel,
+            hyperalignment=params.hyperalignment,
+            full_matrix=params.combine_neighbormappers,
+            use_same_features=params.use_same_features,
+            exclude_from_model=params.exclude_from_model,
+            dtype=params.dtype)
+
         # Performing SL processing manually
```

Comment:
It shouldn't... I will check when get to the laptop

On March 28, 2016 1:46:26 PM EDT, Swaroop Guntupalli <notifications@github.com> wrote:
>>          # Setting up SearchlightHyperalignment
>>          # we need to know which original features where comprising
>the
>>          # individual SL ROIs
>>          _shpaldebug('Initializing HyperalignmentMeasure.')
>> -        hmeasure = HyperalignmentMeasure(featsel=params.featsel,
>> -                                        
>hyperalignment=params.hyperalignment,
>> -                                        
>full_matrix=params.combine_neighbormappers,
>> -                                        
>use_same_features=params.use_same_features,
>> -                                        
>exclude_from_model=params.exclude_from_model, dtype=params.dtype)
>> +        hmeasure = HyperalignmentMeasure(
>> +            featsel=params.featsel,
>> +            hyperalignment=params.hyperalignment,
>> +            full_matrix=params.combine_neighbormappers,
>> +            use_same_features=params.use_same_features,
>> +            exclude_from_model=params.exclude_from_model,
>> +            dtype=params.dtype)
>> +
>>          # Performing SL processing manually
>
>why does my PyCharm keep saying this is underindented? Is it not PEP8?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/swaroopgj/PyMVPA/pull/3/files/e2f1ebea798f5d5a8a09625f4b1bd79b5f92dda1#r57603551